### PR TITLE
Added: navigation and list

### PR DESCRIPTION
**Pull request by aiman-ansari**
Screenshot (21)
Screenshot (22)
Screenshot (23)
Screenshot (24)
Screenshot (25)

### CODE REVIEW

File: src/components/Navigation/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 350px; 
+    height:38px;
+    margin-right: 1rem;
+     background-color:var(--white); 
+    border-radius: 20px;
+     padding:0px 6px;
```
Comment (Ananyamadhu08) [inline src/components/Navigation/Navigation.css:59]: use rem instead of px

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 🚀

File: src/components/Navigation/Navigation.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    color:var(--dark-gray-text);
+    font-size: 0.9rem;
+    cursor: pointer;
+}
+.searchbox-light,.searchbox{
+    display: flex;
+    padding:1px 1rem ;
+    align-items: center;
+    border: 2px solid var(--dark-gray-text);
+    border-radius: 3rem;
```
Comment (deekshithmd) [inline src/components/Navigation/Navigation.css:89]: for border-radius better to use px instead of rem

**Review by deekshithmd — APPROVED** [review]

**Review by saurabhchirde — APPROVED** [review]
Clean and understandable code with proper naming conventions. 
Good to merge.

**Pull request merged**